### ISSUE TITLE
Cannot bind argument to parameter 'SearchPattern' because it is an empty string

### ISSUE THREAD

**Issue by danroth27** [jakobehn/VSTSGitHubTasks#15]
I get this powershell failure if I leave the assets field blank:

```
Cannot bind argument to parameter 'SearchPattern' because it is an empty string
```

**Comment by jdshkolnik** [comment]
@jakobehn I'm hitting this same error. It was working after your changes in #11 but it seems something in the commit from #12 broke it again.

**Comment by jdshkolnik** [comment]
@jakobehn Any update on this?

**Comment by jdshkolnik** [comment]
FYI @jessehouwing

**Comment by jakobehn** [comment]
Sorry for dead slow response.. This issue is fixed in the latest release (0.11.1)

**Comment by jdshkolnik** [comment]
That did the trick. Thanks.
